content/guides/getstarted/how-to-create-a-token.md

```diff
@@ -233,16 +232,16 @@ you just uploaded:
 {
   "name": "Example Token",
   "symbol": "EXMPL",
-  "description": "Example token for Solana Foundation Bootcamp.",
-  "image": "https://github.com/mikemaccana/token-command-line/blob/main/lights.png"
+  "description": "Example token from Solana Making a Token guide.",
+  "image": "https://github.com/solana-developers/opos-asset/blob/7e9408bccd7611bc7a4c4e6847edd10308b92b5f/assets/CompressedNFT/image.png"
```

Comment:
This needs to be the raw file link so it can be rendered when people copy pasta it